src/main/java/se/sundsvall/contract/api/model/Contract.java

```diff
@@ -1,49 +1,55 @@
 package se.sundsvall.contract.api.model;
 
+import java.time.LocalDate;
 import java.util.List;
 
-import com.fasterxml.jackson.annotation.JsonSubTypes;
-import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import org.geojson.FeatureCollection;
 
 import se.sundsvall.contract.model.ExtraParameterGroup;
+import se.sundsvall.contract.model.Fees;
 import se.sundsvall.contract.model.TermGroup;
 import se.sundsvall.dept44.common.validators.annotation.OneOf;
 
 import io.swagger.v3.oas.annotations.media.ArraySchema;
 import io.swagger.v3.oas.annotations.media.Schema;
 import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
-import lombok.experimental.SuperBuilder;
-
-@JsonTypeInfo(
-	use = JsonTypeInfo.Id.NAME,
-	include = JsonTypeInfo.As.EXISTING_PROPERTY,
-	property = "type"
-)
-@JsonSubTypes({
-	@JsonSubTypes.Type(value = LandLeaseContract.class, name = "LAND_LEASE")
-})
+
 @Data
-@EqualsAndHashCode
-@SuperBuilder(setterPrefix = "with")
+@Builder(setterPrefix = "with")
+@AllArgsConstructor
```

Comment:
Shouldn't this be the other way around? I.e. that `@AllArgsConstructor` is package-protected but the `@NoArgsConstructor` isn't. (Since the probability of anyone/anything except the Lombok builder being prone to use the all-args one is next-to-zero, but the possibility of using the no-args one in conjunction with setters would be feasible)

Reply:
Sounds very reasonable